tests/integration_tests/performance/test_network_ab.py

```diff
@@ -120,55 +131,40 @@ def __init__(self, microvm, mode, host_ip, payload_length):
 
 @pytest.mark.nonci
 @pytest.mark.timeout(3600)
-@pytest.mark.parametrize("vcpus", [1, 2])
+@pytest.mark.parametrize("network_microvm", [1, 2], indirect=True)
 @pytest.mark.parametrize("payload_length", ["128K", "1024K"], ids=["p128K", "p1024K"])
 @pytest.mark.parametrize("mode", ["g2h", "h2g", "bd"])
 def test_network_tcp_throughput(
-    microvm_factory,
-    guest_kernel,
-    rootfs,
-    vcpus,
+    network_microvm,
     payload_length,
     mode,
     metrics,
-):
+):  # pylint:disable=redefined-outer-name
```

Comment:
Honestly, I think we should ignore this one by default. We suppress it all over the place because of fixtures, its a very annoying lint. 

As for this one, I thought if I put it on a line it will only suppress it on that line, did I get that wrong? :o

I don't really like the idea of renaming the fixture, aka lint-driving-development :(

Reply:
+1 I would just ignore this error altogether

Reply:
I'll add a commit to disable it to https://github.com/firecracker-microvm/firecracker/pull/4149 :)